tools/lou_checkyaml.c

```diff
@@ -162,16 +171,58 @@ read_table_query(yaml_parser_t *parser, const char **table_file_name_check) {
 	return query_as_string;
 }
 
-static char *
-read_table(yaml_event_t *start_event, yaml_parser_t *parser, const char *display_table) {
-	char *table = NULL;
-	if (start_event->type != YAML_SCALAR_EVENT ||
-			strcmp((const char *)start_event->data.scalar.value, "table"))
-		return 0;
+static void
+compile_inline_table(const char *table_name, const char *table_content, int location) {
+	char *p = (char *)table_content;
+	char *line_start = p;
+	int line_len = 0;
+	while (*p) {
+		if (*p == 10 || *p == 13) {
+			char *line = strndup((const char *)line_start, line_len);
+			if (!lou_compileString(table_name, line))
+				error_at_line(EXIT_FAILURE, 0, file_name, location, "Error in table %s",
+						table_name);
+			location++;
+			free(line);
+			line_start = p + 1;
+			line_len = 0;
+		} else {
+			line_len++;
+		}
+		p++;
+	}
+}
 
-	table = malloc(sizeof(char) * MAXSTRING);
-	table[0] = '\0';
+static void
+compile_include(const char *table_name, const char *tableList, int location) {
+	char *p;
+	char *includeRule = (char *)malloc(sizeof(char) * MAXSTRING);
+	char *_tableList;
```

Comment:
I thought `_foo` var names were not kosher?

Reply:
Should be fine if it's local to a function, no?

Reply:
According to the C standard names beginning with `_` are reserved, see [this stackoverflow answer](https://stackoverflow.com/a/25090719) and section 7.1.3 of [the C standard](http://www.open-std.org/jtc1/sc22/wg14/www/docs/n1570.pdf).

Reply:
OK renamed it.